lib/src/client/call.dart

```diff
@@ -318,8 +363,12 @@ class ClientCall<Q, R> implements Response {
 
   @override
   Future<void> cancel() {
+    final error = GrpcError.cancelled('Cancelled by client.');
+    _timeline?.finish(arguments: {
+      'cancel': error.toString(),
```

Comment:
I think this should still be an `error` key.

Reply:
Done